DirectXTex/DirectXTexUtil.cpp

```diff
@@ -270,6 +270,10 @@ REFGUID DirectX::GetWICCodec(WICCodecs codec) noexcept
     case WIC_CODEC_ICO:
         return GUID_ContainerFormatIco;
 
+    case WIC_CODEC_HEIF:
+        // This requires installing https://aka.ms/heif
+        return GUID_ContainerFormatHeif;
```

Comment:
This symbol exists in the Windows 10 SDK (19041) which is the oldest SDK I support.